jdk/make/closed/autoconf/openj9ext-version-numbers

```diff
@@ -18,4 +18,4 @@
 # ===========================================================================
 
 JDK_MOD_VERSION=0
-JDK_FIX_VERSION=2
+JDK_FIX_VERSION=0
```

Comment:
Was this supposed to be 0, 1, or 2?  Not sure what the final naming convention for the release was.